.github/workflows/manual-nixos.yml

```diff
@@ -0,0 +1,25 @@
+name: "Build NixOS manual"
+
+on:
+  pull_request_target:
+    paths:
+      - 'nixos/**'
+
+jobs:
+  nixos:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          # pull_request_target checks out the base branch by default
+          ref: refs/pull/${{ github.event.pull_request.number }}/merge
+      - uses: cachix/install-nix-action@v12
+        with:
+          # explicitly enable sandbox
+          extra_nix_config: sandbox = true
+      - uses: cachix/cachix-action@v8
+        with:
+          name: nixpkgs-ci
+          signingKey: '${{ secrets.CACHIX_SIGNING_KEY }}'
+      - name: Building NixOS manual
+        run: nix-build nixos/release.nix -A manual.x86_64-linux
```

Comment:
```suggestion
        run: nix-build --option restrict-eval true nixos/release.nix -A manual.x86_64-linux
```

Reply:
I think both manuals could be build using a job matrix. Most steps except the build command are the same.

Reply:
I don't think the `paths` filter works with job matrix?